rtloader/include/rtloader_types.h

```diff
@@ -125,6 +125,8 @@ typedef void (*cb_set_external_tags_t)(char *, char *, char **);
 typedef void (*cb_write_persistent_cache_t)(char *, char *);
 // (value)
 typedef char *(*cb_read_persistent_cache_t)(char *);
+// (sql_query, error)
```

Comment:
Could we say `// (sql_query, error_message)` ?

Reply:
Renamed